contributed_definitions/nyaml/NXresolution.yaml

```
@@ -0,0 +1,158 @@
+category: base
+doc: |
+  Describes the resolution of a physical quantity.
+type: group
+NXresolution(NXobject):
+  physical_quantity:
+    doc: |
+      The physical quantity of the resolution, e.g.,
+      energy, momentum, time, etc.
+  type:
+    doc: |
+      The process by which the resolution was determined.
+    enumeration: [estimated, derived, calibrated, other]
+  note(NXnote):
+    doc: |
+      Additional details of the estimate or description of the calibration procedure
+  resolution(NX_FLOAT):
+    unit: NX_ANY
+    doc: |
+      The resolution of the physical quantity.
+  resolution_errors(NX_FLOAT):
```

maybe resolution_error (without s)
```suggestion
  resolution_error(NX_FLOAT):
```

---

Changed.

I guess this cannot be multidimensional, right? Like spatial_resolution in two directions?

---

I would say it can be multidimensional, still I would call it error (singular). This is, however, somewhat at conflict with NXdata, so I'm fine with both.
@domna What is your opinion?

---

I think it might be better to keep it as `..._errors`, even though I believe that without the `s` is more fitting to other conventions we use. It is, however, convention to generally call all errors with `variable_errors` in nexus (see https://manual.nexusformat.org/design.html#fields), so I would follow this recommendation if we don't have strong arguments against it.

---

Agreed, I will change it back to `resolution_errors`.

---

@domna thanks for the comment, let's do it like that. Feel free to resolve after changing.

---

I changed it, resolving.